README.md

```diff
@@ -1,20 +1,13 @@
 # SCJ-Online-Booking
-Superior Courts Judiciary Online Booking
+Superior Courts Judiciary Lottery Booking Prototype
 
-![img](https://img.shields.io/badge/Lifecycle-Stable-97ca00)
+## Requirements
 
-## License
```

Comment:
We aren't allowed to remove the Apache license

Reply:
I believe this was was a mistake, I must have accidentally overwritten the base ReadMe, this was supposed to only be in the Prototype project